VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
From the previous PR:

> Btw, I am not sure why this file is here and what it does, do you have more context @microbit-sam ?

Reply:
I think it's only used by the build/deploy scripts so maybe it should be removed from this PR

Reply:
This still means that on python-editor there are two different places we need change the version number, any way it could be merged into a single source of truth? (After syncing repos)

Reply:
Yep absolutely. Could you create a new ticket in the python-editor repo to not forget about this?